hazelcast/src/main/java/com/hazelcast/instance/impl/NodeExtension.java

```diff
@@ -397,6 +399,11 @@ default boolean isClientFailoverSupported() {
      */
     CPPersistenceService getCPPersistenceService();
 
+    /**
+     * Returns CP subsystem implementation.
+     */
+    CPSubsystem getCPSubsystem(NodeEngine nodeEngine);
```

Comment:
align name of method with actual implementation:
```suggestion
    CPSubsystem createCPSubsystem(NodeEngine nodeEngine);
```

Reply:
Done.